logger.js

```diff
@@ -63,15 +65,20 @@ function pinoLogger (opts, stream) {
 
   function onResFinished (err) {
     this.removeListener('error', onResFinished)
+<<<<<<< Updated upstream
+=======
     this.removeListener('finish', onResFinished)
+>>>>>>> Stashed changes
```

Comment:
It seems you have made some git mistakes :). Would you mind to fix them?

Reply:
@dalexanco: ^^

Reply:
@dalexanco feel free to send a fresh PR if you'd like.

Reply:
@mcollina: https://github.com/pinojs/pino-http/pull/123
@dalexanco: I really wanted to get this merged upstream, so, I checked out your branch and committed the fix with you as the co-author; all your other commits are left intact. Thanks for the fix!

Reply:
my bad, I have been away for a while. 
Thank you for the 2nd PR !
Really smart to allow the feature only on v12, that was the real reason of my absence on this PR : I fail to make it work on v10 😅